SpatialGDK/Documentation/content/how-to-use-schema.md

```diff
@@ -6,45 +6,57 @@
+
 ## How to generate schema
 
-There are two ways to generate schema for your project:
+There are two ways to generate schema for your project from [within the Unreal Editor](#generating-schema-within-the-unreal-editor), and you can also generate schema [via the command line](#generating-schema-via-the-command-line).
+
+### Generating schema within the Unreal Editor
+Within the Unreal Editor, you can generate schema using a Full Scan, or generate schema iteratively. For information on when to use each method, see [Which method to use](#which-method-to-use).
 
-* **Generate schema (Full Scan)** 
+* **Full Scan** 
 
-    To generate schema for all classes* in your project:<br/>
+    To generate schema for all classes in your project:<br/>
 
-    In the Unreal Editor, on the [GDK Toolbar]({{urlRoot}}/content/toolbars#buttons), open the **Schema** drop-down menu and select **Schema (Full Scan)**.<br/> You must select **Schema (Full Scan)** the first time you generate schema for a project. 
+    In the Unreal Editor, on the [GDK toolbar]({{urlRoot}}/content/toolbars#buttons), open the **Schema** drop-down menu and select **Schema (Full Scan)**.<br/> You must select **Schema (Full Scan)** the first time you generate schema for a project. 
     <br/> ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/schema-button-full-scan.png)<br/>
     _Image: In the GDK toolbar in the Unreal Editor, select **Schema (Full Scan)**_<br/>
-    <br/>When you select **Schema (Full Scan)**, the GDK automatically iterates through all classes* in your project to generate the schema files, and then updates the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase). <br/>
+    <br/>When you select **Schema (Full Scan)**, the GDK automatically iterates through all classes in your project to generate the schema files, and then updates the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase). <br/>
 
-    Run a full scan the first time you generate schema for your project, and whenever you need to generate schema for classes* that are not currently loaded by the Editor.<br/>For example: You need to select **Schema (Full Scan)** if you didn't generate schema after adding a new Blueprint to your game, and that Blueprint is no longer open in the Editor.<br/><br/>
+    Run a full scan the first time you generate schema for your project, and whenever you need to generate schema for classes that are not currently loaded by the Editor.<br/>For example: You need to select **Schema (Full Scan)** if you didn't generate schema after adding a new Blueprint to your game, and that Blueprint is no longer open in the Editor.<br/><br/>
 
-* **Generate Schema (Iterative)**
+* **Iterative**
 
-    To generate schema for classes* that are currently loaded by the Editor: <br/>
+    To generate schema for classes that are currently loaded by the Editor: <br/>
```

Comment:
What's the benefit of using this rather than full scan? - is it because it's quicker? If so, should we say that?

Reply:
@padhansell do you know?

Reply:
Yep exactly - full scan is significantly slower than iterative in large projects. To me it seems obvious from the naming that the iterative approach would be faster, but agree we should call that out in the "Which method to use" section 👍